server/models/practice.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose')
+
+const practiceSchema = new mongoose.Schema({
+    student: {
+        // look into type, see if mongoose accepts objectID
+        type: String,
+        required: true
+    },
+    rcsid: {
+        type: String,
+        required: true
+    },
+    league: {
+        type: String,
+        required: true
+    },
+    practice_type:{
+        type: String,
+        required: true
+    },
+    results: [{
```

Comment:
This will need to be changed in the future to support flexible questions, but it's good for now.